src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs

```diff
@@ -22,23 +22,27 @@ public static class PrereleaseCompatibilityTransformer
         {
             bool modifiedLog = false;
             updatedLog = null;
-            var settings = new JsonSerializerSettings { Formatting = formatting, DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate };
+            var settings = new JsonSerializerSettings
+            {
+                Formatting = formatting,
+                DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate,
+                DateParseHandling = DateParseHandling.None
+            };
 
             if (string.IsNullOrEmpty(prereleaseSarifLog)) { return null; }
 
-            JObject sarifLog = JObject.Parse(prereleaseSarifLog);
+            JObject logObject = SarifUtilities.DeserializeObject<JObject>(prereleaseSarifLog);
```

Comment:
>logObject [](start = 20, length = 9)

I changed this name because this code was using names like `sarifLog` for three different things:

1. A string containing the JSON contents of a SARIF log file.
2. A `SarifLog` object.
3. A `JObject` or a `JToken` resulting from a Newtonsoft parse of a JSON string.

... and I found that confusing.